pyuvsim/tests/test_source.py

```diff
@@ -9,9 +9,14 @@
 from astropy.time import Time
 from astropy.coordinates import Angle, SkyCoord, EarthLocation
 from astropy import units
+import healpy as hp
```

Comment:
Only tests requiring healpy should import healpy and those tests should be skipped if healpy is not installed.